### DISCUSSION TITLE
Merging

### DISCUSSION THREAD

**Discussion by farzanehnematpour** [farzanehnematpour/Tafe24S1_14#2] [General]
I am creating this topic just for the next step of the assessment as it needs us to show disccussions on three different topics

**Comment by farzanehnematpour** [comment]
I will be in touch for the rest of the assessment

**Comment by OscarCastle573** [comment]
Alright, sorry that I'm messing stuff up

**Comment by farzanehnematpour** [comment]
I am creating Mortgage branch as the feature and you go with Currency branch. Make sure checkout development branch first.
